File: examples-sdk/java/035-basic-vil-service-module/Main.java
// 035-basic-vil-service-module — Java SDK equivalent
// Compile: vil compile --from java --input 035-basic-vil-service-module/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}

File: examples-sdk/csharp/035-basic-vil-service-module/Main.cs
// 035-basic-vil-service-module — C# SDK equivalent
// Compile: vil compile --from csharp --input 035-basic-vil-service-module/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();
